OpenLIFUData/OpenLIFUData.py

```diff
@@ -273,6 +276,16 @@ def onLoadProtocolPressed(self) -> None:
         if filepath:
             self.logic.load_protocol(filepath)
 
+    def onLoadVolumePressed(self) -> None:
+        """ Call slicer dialog to load volumes into the scene"""
+        return slicer.util.openAddVolumeDialog()
+    
+    def onLoadFiducialsPressed(self) -> None:
+        """ Call slicer dialog to load fiducials into the scene"""
+        ioManager = slicer.app.ioManager()
+        return ioManager.openDialog("MarkupsFile", slicer.qSlicerFileDialog.Read)
```

Comment:
can you add a code comment here indicating that we should use `slicer.util.whateverItWasCalled` but that we are doing this workaround for such and such reason